test_definitions.py

```diff
@@ -114,6 +118,25 @@ def test_filename_matches_type_version_fields(definition_file):
     assert version == definition_file.definition["_version"]
 
 
+@pytest.mark.parametrize(
+    "definition_file",
+    EVENT_DEFINITIONS + OTHER_DEFINITIONS,
+    ids=EVENT_DEFINITION_IDS + OTHER_DEFINITION_IDS,
+)
+def test_schema_validation(definition_file):
+    # If you try to just add on to the meta schema it doesn't work see this for more info
+    # https://stackoverflow.com/questions/54719010/how-do-you-extend-json-schema-meta-schema-to-support-new-properties
+    #
+    # You should be able to create a vocabulary, but I have not managed. See the following for more
```

Comment:
Do we want to do it properly or go with this crude version?

Reply:
I'm okay with this. Not perfect but it's an improvement.